MonoGame.Framework/Input/GamePadCapabilities.cs

```diff
@@ -46,6 +46,10 @@ public struct GamePadCapabilities
     {
         public bool IsConnected { get; internal set; }
 
+        public string DisplayName { get; internal set; }
+
+        public string Id { get; set; }

```

Comment:
On consoles you can set gamepad ids to 1, 2, 3, 4, they don't need to be real ids.